cypress/e2e/functional/document_tests/tiles_copy_test_spec.js

```diff
@@ -324,6 +324,40 @@ context('Test copy tiles from one document to other document', function () {
     testPrimaryWorkspace2();
 
   });
+  it('should enable/disable copy-to-workspace and copy-to-document buttons appropriately', function () {
+    beforeTest(student5);
+    cy.openTopTab('problems');
+    cy.openProblemSection('Initial Challenge');
+
+    // Verify initial disabled state
+    cy.get('[data-testid="tool-copytoworkspace"], [data-testid="tool-copytodocument"]')
+      .should('have.class', 'disabled');
```

Comment:
I'd like to have a check that it is _actually_ disabled - that is, doesn't do anything when clicked - as well as just having that visual class.

Reply:
Also, since there are two class attributes in play, should include the negative tests in each.
```suggestion
      .should('have.class', 'disabled')
      .should('not.have.class', 'enabled');
```